File: backend/candidate-profile-service/src/main/java/com/jobportal/candidateprofile/dto/StateDto.java
package com.jobportal.candidateprofile.dto;

public class StateDto {

    private Integer sid;
    private String sname;

    public StateDto() {
    }

    public StateDto(Integer sid, String sname) {
        this.sid = sid;
        this.sname = sname;
    }

    public Integer getSid() {
        return sid;
    }

    public void setSid(Integer sid) {
        this.sid = sid;
    }

    public String getSname() {
        return sname;
    }

    public void setSname(String sname) {
        this.sname = sname;
    }
}

File: backend-dotnet/Services/JobPortal.JobApplicationService/DTOs/StateDto.cs
namespace JobPortal.JobApplications.DTOs;

public class StateDto
{
    public int? Sid { get; set; }
    public string? Sname { get; set; }

    public StateDto() { }

    public StateDto(int? sid, string? sname)
    {
        Sid = sid;
        Sname = sname;
    }
}
